src/components/common/TmSessionSignUp.vue

```diff
@@ -214,36 +210,48 @@ export default {
       }
     }
   },
+  created() {
+    this.steps = [
```

Comment:
This defines the steps order and their slot names
It also defines the validation per step to do.

Reply:
Nice thing is, this parent component still keeps the whole form and data in one place and can handle the whole data.